addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -565,3 +565,28 @@ def test_export_invoice_with_two_downpayments(self):
         )
         invoice.action_post()
         self._assert_export_invoice(invoice, 'test_export_invoice_with_two_downpayments.xml')
+
+    def test_send_prezzo_unitario_converted_to_company_currency(self):
```

Comment:
We need to use the `@freeze_time` decorator here, otherwise `DataScadenzaPagamento` is going to change everyday :wink: